### PR TITLE
Update graph colours

### PR DESCRIPTION
**Pull request by rstrange-scottlogic**

### CODE REVIEW

File: air-quality-ui/src/components/single-city/SingleCity.tsx [hunk 1]
```diff
@@ -163,10 +163,12 @@ export const SingleCity = () => {
...
   useEffect(() => {
     const updateColors = async () => {
+      let index = 0
       const colorsBySite: Record<string, string> = {}
       for (const { value } of sites) {
-        const color = await textToColor(value)
+        const color = await indexToColor(index)
         colorsBySite[value] = color
+        index++
       }
```
Comment (lincent) [inline air-quality-ui/src/components/single-city/SingleCity.tsx:172]: could you not make use of the following? 

sites.foreach((value, index) => {})

File: air-quality-ui/src/components/single-city/SingleCity.module.css [hunk 2]
```diff
@@ -34,6 +34,7 @@
     align-content: center;
     width: 600px;
     height: 400px;
+    text-align: center;
```
Comment (lincent) [inline air-quality-ui/src/components/single-city/SingleCity.module.css:37]: what does this do?

File: air-quality-ui/src/services/echarts-service.ts [hunk 3]
```diff
@@ -31,30 +30,24 @@ export const convertToLocalTime = (date: string): string | null => {
...
-      .map((byte) => byte.toString(16).padStart(2, '0'))
-      .join('')
-  }
-
-  // Main function to generate the color
-  return generateHash(text).then((hash) => {
-    // Use the first 6 characters of the hash to form the color code
-    return `#${hashToHex(hash).slice(0, 6)}`
-  })
+export const indexToColor = async (index: number): Promise<string> => {
```
Comment (lincent) [inline air-quality-ui/src/services/echarts-service.ts:37]: I think you can stop this being asynchronous?

File: air-quality-ui/src/components/single-city/SingleCity.module.css (hunk 2 above)
Comment (rstrange-scottlogic) [reply air-quality-ui/src/components/single-city/SingleCity.module.css:37]: Makes the loading spinners align to the middle of the div when loading the colours/lines takes a bit longer. Previously they were left aligned which looked a bit weird

File: air-quality-ui/src/components/single-city/SingleCity.tsx (hunk 1 above)
Comment (rstrange-scottlogic) [reply air-quality-ui/src/components/single-city/SingleCity.tsx]: Yes, much nicer now, thanks!

File: air-quality-ui/src/components/single-city/SingleCity.module.css (hunk 2 above)
Comment (rstrange-scottlogic) [reply air-quality-ui/src/components/single-city/SingleCity.module.css]: Actually it did something bizarre to the text I've just noticed so going to revert and try something else

File: air-quality-ui/src/services/echarts-service.ts (hunk 3 above)
Comment (rstrange-scottlogic) [reply air-quality-ui/src/services/echarts-service.ts]: Yep, good spot thanks

**Review by lincent — APPROVED** [review]

**Pull request merged**